build.sbt

```diff
@@ -55,7 +55,7 @@ sourceGenerators in Compile ++= Seq(
 )
 
 // Lib dependencies
-val sonarVersion = "8.3.0.34182"
+val sonarVersion = "8.4.2.36762"
```

Comment:
I'll downgrade this version to `8.4.0.35506` to support the whole `8.4.x` series, but I want to test whether that will actually work also agains SonarQube 8.4.2. Apologies for the delay.

Reply:
No problem!